workflow/workflow.go

```diff
@@ -21,7 +21,6 @@ const (
 type Workflow struct {
 	Version          string            `yaml:"workflowVersion"`
 	SpeakeasyVersion Version           `yaml:"speakeasyVersion,omitempty"`
-	VersionLocked    bool              `yaml:"versionLocked,omitempty"`
```

Comment:
What was this being used for initially? Was it something you added?

Reply:
Yes it was, now no longer needed